metron-platform/metron-hbase/src/test/java/org/apache/metron/hbase/mock/MockHTable.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.metron.test.mock;
+package org.apache.metron.hbase.mock;
```

Comment:
Part of the upstream changes during the merge. Even though it's not mine, GitHub wants to show it as my change (because there was a conflict here I resolved).